docs/users/standalone/README.md

```diff
@@ -0,0 +1,174 @@
+
+What Gradle does is equivalent to running these commands
+```
+mkdir build
+cp -r core/nodejsActionBase/* build
+cp core/nodejs18Action/Dockerfile build
+```
+
+Now, run the `distDocker` command to generate a local Docker image for the chosen runtime version. (Make sure docker daemon is running)
+
+```
+./gradlew core:nodejs18Action:distDocker
+```
+
+This will return the following runtime image with the name `action-nodejs-v18`. Since a docker image is created, you can check the `IMAGE ID` for `nodejs-action-v18`
+```
+docker images
+```
+## Running the Container
+For the testing purpose, we are going to start the container locally that has Node.js app server inside. The Apache OpenWhisk platform uses this server to inject action code into the runtime and fire invocation requests. In order to access this service within the container from the outside, as we are about to do using `curl`, port mapping needs to be done next. As a result, we can now access the web service inside docker by first reaching an IP port on `localhost`, which subsequently forwards the request to the docker container's designated port.
+In our example, the `Action` container exposes `port 8080` (see the Dockerfile for the associated Docker image), thus we publish the container's `port 8080` to the `localhost` (here, `port 3008` on `localhost` is chosen arbitrarily, as long as the port is not already used for something else):
+```
+docker run --publish 3008:8080 --name=bloom_whisker -i -t action-nodejs-v18:latest
+```
+A simpler way is to map `port 80` on `localhost` to the container's `port 8080`. The port number assigned to the HTTP protocol is `80`. Since we will be sending actions against the runtime using HTTP, using this number will allow us to omit the port in the request later. Oftentimes, `port 80` could already be occupied by another process. Without loss of generality, the following examples will use the arbitrarily chosen `port 3008`.
+
+Lists all running containers
+```
+docker ps
+```
+or
+```
+docker ps -a
+```
+You should see a container named `bloom_whisker` being run.
+
+## Create your function
+A container can only hold one function. This first example prepared a `js-init.json` file which contains the function.
+```json
+{
+  "value": {
+    "name" : "nodejs-helloworld",
+    "main" : "main",
+    "binary": false,
+    "code" : "function main() {return {payload: 'Hello World!'};}"
+  }
+}
+```
+The json file contains a simple JavaScript (the target runtime language) function, which is the actual payload.
+
+## Initialze the Runtime
+Before issuing the action against the runtime, we first initialize the function with by invoking the ```/init``` endpoint.
+```
+curl -H "Content-Type:application/json" -X POST --data '@/$FILEPATH/js-init.json' http://localhost:3008/init
```

Comment:
Please eliminate the $FILEPATH as it adds complexity to each example.  Instead, formulate examples using the actual test files relative to where a developer's clone of the repo. would be.